resources/lambda/browsershot.js

```diff
@@ -47,8 +48,39 @@ exports.handle = async function (event) {
     // If there was a path, then read the file and return it.
     if (event.options.path) {
         let contents = fs.readFileSync(event.options.path);
+
         fs.unlinkSync(event.options.path);
 
+        // If the file destination is S3, then write
+        // the file and return the ETag as confimation.
+        if (event.options.s3) {
+            const s3 = new AWS.S3({ region: event.options.s3.region });
```

Comment:
In addition to making the S3 options configurable, I suggest accepting `key` and `secret` as an option and passing them in when present:

```suggestion
            const accessKeyId = event.options.s3.key;
            const secretAccessKey = event.options.s3.secret;
            const s3 = new AWS.S3({
                region: event.options.s3.region,
                ...(accessKeyId && secretAccessKey ? { accessKeyId, secretAccessKey } : {}),
            });
```

Reply:
Updated my comment. If the key and secret configured for the disk works for the Laravel app, then there should be no issues with the lambda using it as a credential.